src/test/java/seedu/address/logic/parser/TrackrParserTest.java

```diff
@@ -13,58 +13,122 @@
+        AddTutorialGroupCommand command = (AddTutorialGroupCommand) parser.parseCommand((
+                TutorialGroupUtil.getAddTutorialGroupCommand(tutorialGroup)));
+        assertEquals(new AddTutorialGroupCommand(tutorialGroup), command);
+    }
+
+    @Test
+    public void parseCommand_addStudent() throws Exception {
+        Student student = new StudentBuilder().build();
+        AddStudentCommand command = (AddStudentCommand) parser.parseCommand(StudentUtil.getAddStudentCommand(student));
+        assertEquals(new AddStudentCommand(student), command);
+    }
+
+    @Test
+    public void parseCommand_addAttendance() throws Exception {
+        Attendance attendance = new AttendanceBuilder().build();
+        AddAttendanceCommand command =
+            (AddAttendanceCommand) parser.parseCommand(AttendanceUtil.getAddAttendanceCommand(attendance));
+        assertEquals(new AddAttendanceCommand(INDEX_FIRST_PERSON, new int[]{2}), command);
+    }
+
     @Test
     public void parseCommand_clear() throws Exception {
         assertTrue(parser.parseCommand(ClearCommand.COMMAND_WORD) instanceof ClearCommand);
         assertTrue(parser.parseCommand(ClearCommand.COMMAND_WORD + " 3") instanceof ClearCommand);
     }
 
-    //    @Test
-    //    public void parseCommand_delete() throws Exception {
-    //        DeleteCommand command = (DeleteCommand) parser.parseCommand(
-    //                DeleteCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased());
-    //        assertEquals(new DeleteCommand(INDEX_FIRST_PERSON), command);
-    //    }
-
     @Test
     public void parseCommand_deleteModule() throws Exception {
         DeleteModuleCommand command = (DeleteModuleCommand) parser.parseCommand(
                 DeleteModuleCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased());
         assertEquals(new DeleteModuleCommand(INDEX_FIRST_PERSON), command);
     }
 
+    //    @Test
+    //    public void parseCommand_deleteTutorialGroup() throws Exception {
+    //        DeleteTutorialGroupCommand command = (DeleteTutorialGroupCommand) parser.parseCommand(
+    //                DeleteTutorialGroupCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased());
+    //        assertEquals(new DeleteTutorialGroupCommand(INDEX_FIRST_PERSON), command);
+    //    }
+
+    @Test
+    public void parseCommand_deleteStudent() throws Exception {
+        DeleteStudentCommand command = (DeleteStudentCommand) parser.parseCommand(
+                DeleteStudentCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased());
+        assertEquals(new DeleteStudentCommand(INDEX_FIRST_PERSON), command);
+    }
+
+    @Test
+    public void parseCommand_deleteAttendance() throws Exception {
+        DeleteAttendanceCommand command = (DeleteAttendanceCommand) parser.parseCommand(
+                DeleteAttendanceCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased()
+                + " " + "week/2");
```

Comment:
You can put the " " with the "week/2" as one String.

Reply:
Thanks for spotting this. Have rectified it and the rest below.